helix-core/src/indent.rs

```diff
@@ -388,6 +417,23 @@ fn query_indents(
                         }
                     }
                 }
+                "one-line?" | "not-one-line?" => match pred.args.get(0) {
+                    Some(QueryPredicateArg::Capture(capture_idx)) => {
+                        let node = m.nodes_for_capture_index(*capture_idx).next();
+
+                        match node {
+                            Some(node) => {
+                                let (start_line, end_line) = (get_line_num(node, End::Start), get_line_num(node, End::End));
+                                let one_line = end_line == start_line;
+                                !(one_line ^ (pred.operator.as_ref() == "one-line?"))
```

Comment:
This is a bit confusing. I think the fillkwng is clearer. (For bool XOR is just neq)
```suggestion
                                one_line != (pred.operator.as_ref() == "not-one-line?")
`